api/app/controllers/files_controller.rb

```diff
@@ -7,6 +7,7 @@ def show
 
     render(json: {
       filepath: source_file.filepath,
+      filetype: FileClassifier.new(source_file.filepath).filetype,
```

Comment:
Eventually, we should store this directly on the `source_files` row. 
This way we can query all source file of a specific type.

Reply:
👍